src/VideoToolbox/VTCompressionProperties.cs

```diff
@@ -508,7 +511,7 @@ public partial class VTCompressionProperties {
 		[Introduced (PlatformName.TvOS, 13,0)]
 #endif
 		public VTAlphaChannelMode AlphaChannelMode {
-			get => VTAlphaChannelModeExtensions.GetValue (GetNSStringValue (VTCompressionPropertyKey.AlphaChannelMode));
+			get => VTAlphaChannelModeExtensions.GetValue (GetNSStringValue (VTCompressionPropertyKey.AlphaChannelMode)!);
```

Comment:
Why not setting the property as nullable?

Reply:
I applied this change but reverted it due to Rolf's comment below